OpenSim/Simulation/Model/PrescribedForce.cpp

```diff
@@ -42,14 +42,13 @@ using namespace std;
 /**
  * Default constructor.
  */
-PrescribedForce::PrescribedForce(Body* body)
+PrescribedForce::PrescribedForce(const PhysicalFrame* frame)
 {
     setNull();
-    constructProperties();
+    constructInfrastructure();
 
-    _body = body;
-    if (_body)
-        setBodyName(_body->getName());
+    if (frame)
+        updConnector<PhysicalFrame>("frame").setConnecteeName(frame->getName());
```

Comment:
this should also be changed to:
`updConnector<PhysicalFrame>("frame").connect(frame);`
connect will determine the path name to store with the Connector.